src/Entity/MessageTemplate.php

```diff
@@ -282,7 +305,9 @@ public static function create(array $values = []) {
    * {@inheritdoc}
    */
   public function preSave(EntityStorageInterface $storage) {
-    $this->text = array_filter($this->text);
+    $this->text = array_filter($this->text, function ($a) {
```

Comment:
I'm not totaly clear what we do here. also I think `$a` should get a more descriptive name. maybe `$delta`, or `$partial`?